pyproject.toml

```diff
@@ -1,18 +1,21 @@
 [build-system]
-requires = ["flit_core", 'pandas']
```

Comment:
`pandas` is not a build requirement, but a runtime requirement

Reply:
thanks for the heads up